_articles/miscellaneous/attachments.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: article
+title: How-to attach files to an entry
+categories: [miscellaneous]
+featured: false
+popular: false
+tags: [attachments, entry, storage, premium]
+---
+## Overview
+*Note: Support for attachments is a **PREMIUM ONLY** feature.*
```

Comment:
This should go above "Overview" and can use the notes tag for proper formatting:

```
{% note %}
Support for attachments is a **premium** feature.
{% endnote %}
```